openpype/widgets/message_window.py

```diff
@@ -28,23 +32,32 @@ def _info(self):
             self.exit()
 
     def _warning(self):
-        self.setWindowTitle(self.title)
         rc = QtWidgets.QMessageBox.warning(
             self, self.title, self.message)
         if rc:
             self.exit()
 
     def _critical(self):
-        self.setWindowTitle(self.title)
         rc = QtWidgets.QMessageBox.critical(
             self, self.title, self.message)
         if rc:
             self.exit()
 
+    def _ask(self):
+        self.answer = None
+        rc = QtWidgets.QMessageBox.question(
```

Comment:
Is there a  reason we choose `ask` over `question` even though the Qt API method is called `question`? I don't care too much. I just noticed the others do match the Qt counterpart and was wondering.

Reply:
Yeah, think we should change it to `_question`.